File: MakoCelo/frmAbout.cs
using System;
using System.Collections.Generic;
using System.Net;
using System.Net.Mail;
using System.Windows.Forms;
using Microsoft.VisualBasic;

namespace MakoCelo
{
    public partial class frmAbout
    {
        public frmAbout()
        {
            InitializeComponent();
            _Button1.Name = "Button1";
        }

        // Public synth As New Speech.Synthesis.SpeechSynthesizer
        // Public WithEvents recognizer As New Speech.Recognition.SpeechRecognitionEngine
        // Dim gram As New System.Speech.Recognition.DictationGrammar()

        private void frmAbout_Load(object sender, EventArgs e)
        {
            rtbHelp.Rtf = My.Resources.Resources.MakoCELO_HelpFile_0446;
        }

        private void frmAbout_Shown(object sender, EventArgs e)
        {
            // R4.00 Textbox defaults to all text selected.
            // tbHelp.SelectionLength = 0
        }

        private void frmAbout_KeyDown(object sender, KeyEventArgs e)
        {

            // R4.10 KEY PREVIEW is on, check for ESCAPE or RETURN to exit.
            if ((int)e.KeyCode == 27 | (int)e.KeyCode == 13)
                Close();
        }

        private void Button1_Click(object sender, EventArgs e)
        {
            var Recipients = new List<string>();
            Recipients.Add("Receiver@hotmail.com");
            string FromEmailAddress = "Sender@hotmail.com";
            string Subject = "This email came from MakoCELO.";
            string Body = "This code is not implemented but here for future possible use.";
            string UserName = "Sender@hotmail.com";
            string Password = "";
            int Port = 587;
            string Server = "smtp.live.com";
            var Attachments = new List<string>();
            Interaction.MsgBox(SendEmail(Recipients, FromEmailAddress, Subject, Body, UserName, Password, Server, Port, Attachments));
        }

        public string SendEmail(List<string> Recipients, string FromAddress, string Subject, string Body, string UserName, string Password, string Server = "smtp.live.com", int Port = 587, List<string> Attachments = null)
        {
            var Email = new MailMessage();
            try
            {
                var SMTPServer = new SmtpClient();
                foreach (string Attachment in Attachments)
                    Email.Attachments.Add(new Attachment(Attachment));
                Email.From = new MailAddress(FromAddress);
                foreach (string Recipient in Recipients)
                    Email.To.Add(Recipient);
                Email.Subject = Subject;
                Email.Body = Body;
                SMTPServer.Host = Server;
                SMTPServer.Port = Port;
                SMTPServer.Credentials = new NetworkCredential(UserName, Password);
                SMTPServer.EnableSsl = true;
                SMTPServer.Send(Email);
                Email.Dispose();
                return "Email to " + Recipients[0] + " from " + FromAddress + " was sent.";
            }
            catch (SmtpException ex)
            {
                Email.Dispose();
                return ex.Message;
            }
            // Return "Sending Email Failed. Smtp Error."

            catch (ArgumentOutOfRangeException ex)
            {
                Email.Dispose();
                return "Sending Email Failed. Check Port Number.";
            }
            catch (InvalidOperationException Ex)
            {
                Email.Dispose();
                return "Sending Email Failed. Check Port Number.";
            }
        }
    }
}

File: MakoCelo/frmAbout.vb
Imports System.Net
Imports System.Net.Mail
Imports System.Speech.Synthesis

Public Class frmAbout

  'Public synth As New Speech.Synthesis.SpeechSynthesizer
  'Public WithEvents recognizer As New Speech.Recognition.SpeechRecognitionEngine
  'Dim gram As New System.Speech.Recognition.DictationGrammar()

  Private Sub frmAbout_Load(sender As Object, e As EventArgs) Handles MyBase.Load

    rtbHelp.Rtf = My.Resources.MakoCELO_HelpFile_0446

  End Sub

  Private Sub frmAbout_Shown(sender As Object, e As EventArgs) Handles Me.Shown
    'R4.00 Textbox defaults to all text selected.
    'tbHelp.SelectionLength = 0
  End Sub

  Private Sub frmAbout_KeyDown(sender As Object, e As KeyEventArgs) Handles Me.KeyDown

    'R4.10 KEY PREVIEW is on, check for ESCAPE or RETURN to exit.
    If (e.KeyCode = 27) Or (e.KeyCode = 13) Then Me.Close()

  End Sub

  Private Sub Button1_Click(sender As Object, e As EventArgs) Handles Button1.Click

    Dim Recipients As New List(Of String)
    Recipients.Add("Receiver@hotmail.com")
    Dim FromEmailAddress As String = "Sender@hotmail.com"
    Dim Subject As String = "This email came from MakoCELO."
    Dim Body As String = "This code is not implemented but here for future possible use."
    Dim UserName As String = "Sender@hotmail.com"
    Dim Password As String = ""
    Dim Port As Integer = 587
    Dim Server As String = "smtp.live.com"
    Dim Attachments As New List(Of String)
    MsgBox(SendEmail(Recipients, FromEmailAddress, Subject, Body, UserName, Password, Server, Port, Attachments))

  End Sub

  Function SendEmail(ByVal Recipients As List(Of String), ByVal FromAddress As String, ByVal Subject As String, ByVal Body As String, ByVal UserName As String, ByVal Password As String, Optional ByVal Server As String = "smtp.live.com", Optional ByVal Port As Integer = 587, Optional ByVal Attachments As List(Of String) = Nothing) As String

    Dim Email As New MailMessage()

    Try
      Dim SMTPServer As New SmtpClient
      For Each Attachment As String In Attachments
        Email.Attachments.Add(New Attachment(Attachment))
      Next
      Email.From = New MailAddress(FromAddress)
      For Each Recipient As String In Recipients
        Email.To.Add(Recipient)
      Next
      Email.Subject = Subject
      Email.Body = Body
      SMTPServer.Host = Server
      SMTPServer.Port = Port
      SMTPServer.Credentials = New System.Net.NetworkCredential(UserName, Password)
      SMTPServer.EnableSsl = True
      SMTPServer.Send(Email)
      Email.Dispose()
      Return "Email to " & Recipients(0) & " from " & FromAddress & " was sent."

    Catch ex As SmtpException
      Email.Dispose()
      Return ex.Message
      '    Return "Sending Email Failed. Smtp Error."

    Catch ex As ArgumentOutOfRangeException
      Email.Dispose()
      Return "Sending Email Failed. Check Port Number."

    Catch Ex As InvalidOperationException
      Email.Dispose()
      Return "Sending Email Failed. Check Port Number."

    End Try

  End Function


End Class